tests/filters.test.js

```diff
@@ -41,5 +41,99 @@ describe(__filename, () => {
       expect(betterSafe(null)).toEqual(undefined);
       expect(betterSafe('')).toEqual(undefined);
     });
+
+    it('removes meta tags by default', () => {
+      const markup =
+        '<meta content="summary_large_image" name="twitter:card" some="attr">';
+      const betterSafe = makeBetterSafe({
+        markAsSafe(content) {
+          return content;
+        },
+      });
+
+      expect(betterSafe(markup)).toEqual('');
+    });
+
+    it('allows meta tags when isHeadMarkup is true', () => {
+      const markup =
+        '<meta content="summary_large_image" name="twitter:card" some="attr">';
+      const betterSafe = makeBetterSafe({
+        markAsSafe(content) {
+          return content;
+        },
+      });
+
+      expect(betterSafe(markup, { isHeadMarkup: true })).toEqual(
+        '<meta content="summary_large_image" name="twitter:card">'
+      );
+    });
+
+    it('accepts no tags when isHeadMarkup is true', () => {
+      const markup = '<!-- no content -->';
+      const betterSafe = makeBetterSafe({
+        markAsSafe(content) {
+          return content;
+        },
+      });
+
+      expect(betterSafe(markup, { isHeadMarkup: true })).toEqual('');
+    });
+
+    it('accepts property values starting with "og:"', () => {
+      const markup =
+        '<meta content="Mozilla Add-ons Blog" property="og:site_name">';
```

Comment:
Would be good to add a test to cover trying to break out of the quotes and creating a new attribute.

Reply:
Do you have a specific example in mind?

Reply:
Can be anything that tests the value is sanitized.